src/DecisionProgramming.jl

```diff
@@ -26,14 +26,15 @@ export Node,
     validate_influence_diagram
 
 export DecisionVariables,
-    PathProbabilityVariables,
+    PathCompatibilityVariables,
     ForbiddenPath,
-    probability_cut,
-    active_paths_cut,
+    lazy_probability_cut,
     PositivePathUtility,
+    NegativePathUtility,
     expected_value,
     value_at_risk,
-    conditional_value_at_risk
+    conditional_value_at_risk,
+    validate_model
```

Comment:
The function validate_model was removed, so it should be removed from here as well